buildSrc/src/main/java/org/springframework/security/CheckExpectedBranchVersionPlugin.java

```diff
@@ -36,38 +44,67 @@ public void apply(Project project) {
 		TaskProvider<CheckExpectedBranchVersionTask> checkExpectedBranchVersionTask = project.getTasks().register("checkExpectedBranchVersion", CheckExpectedBranchVersionTask.class, (task) -> {
 			task.setGroup("Build");
 			task.setDescription("Check if the project version matches the branch version");
+			task.onlyIf("skipCheckExpectedBranchVersion property is false or not present", CheckExpectedBranchVersionPlugin::onlyIfCheck);
```

Comment:
So, in Develocity this will read:

> Skip reason	'skipCheckExpectedBranchVersion property is false or not present' not satisfied

The double negative is confusing, but we can't change how it appears in Develocity. In code it reads well:

> [run] only if skipCheckExpectedBranchVersion property is false or not present

Reply:
That is a good point. I am not sure how to phrase it to make it readable in both cases.